src/pages/Authentication.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 
-import Navbar from '../components/presentationals/Navbar.jsx';
-import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration.jsx';
-import AuthenticationCard from '../components/presentationals/AuthenticationCard.jsx';
+import Navbar from '../components/presentationals/Navbar/Navbar';
+import AuthenticationIllustration from '../components/presentationals/AuthenticationIllustration/AuthenticationIllustration.jsx';
+import AuthenticationCard from '../components/presentationals/AuthenticationCard/AuthenticationCard';
 import './Authentication.scss';
 
-const AuthenticationLayout = () => {
+const AuthenticationLayout = defaultAvatarUrl => {
```

Comment:
This should be `{defaultAvatarUrl}` instead.

Reply:
this has been resolved